### PR TITLE
Read all reviews

### PR DESCRIPTION
**Pull request by hanien**

### CODE REVIEW

File: server/controllers/reviews.controller.js [hunk 1]
```diff
@@ -52,6 +52,32 @@ router.get('/admin/:id', function(req, res, next) {
   });
 });
 
+// GET / User / Read all reviews
+router.get('/', function(req, res, next) {
+  Review.find(function(err, reviews) {
+    if (err) { return next(err); }
```
Comment (Georgesarkis) [inline server/controllers/reviews.controller.js]: i dont think that a good implementation. i think that should be getting review only base on specific movie id. so when asking for get all reviews it should be get all reviews of a specific movie not for all movies. and the router for the review should always have :id in it

File: server/controllers/reviews.controller.js [hunk 2]
```diff
@@ -52,6 +52,32 @@ router.get('/admin/:id', function(req, res, next) {
...
+    }
+    res.json({"data": reviews});
+    res.status(200);
+  });
+});
+
+// GET / Admin / Read all reviews
+router.get('/admin/', function(req, res, next) {
+  Review.find(function(err, reviews) {
+    if (err) { return next(err); }
```
Comment (Georgesarkis) [inline server/controllers/reviews.controller.js]: same in here maybe it should be find({movie: id}, funciton.... i think that will be much better

File: server/controllers/reviews.controller.js (hunk 1 above)
Comment (hanien) [reply server/controllers/reviews.controller.js]: fixed

**Comment by Georgesarkis** [comment]
just small point. since you have change the router in here and now it's /id/review i think you should delete router.use(./api/review) in index.js otherwise it will not make any sense because the path is right now ./api/review/id(which is id for movie not review)/reivew

**Pull request merged**